Iron-code/node-todo-api
# setup test for project

#### Checklist:


- [ ] All test are passing. 
- [ ] Current changes in the pr have the necessary test.
- [ ] Api Documentation follows api standards.
- [ ] Your commit messages format follows our commit message guideline.


#### What does this pr do?

Setup test for project. Allows us to add test cases as we progress with the project.

closes #7

#### How should this be manually tested?

run `npm install`
1. run `npm run test`
2. Notice the tests passes. I have attached an image to show an example.



##### Any background context you want to provide?


#### What are the relevant issues?

#7 


#### Screenshots (if appropriate)

<img width="338" alt="Screenshot 2020-02-28 at 2 39 30 AM" src="https://user-images.githubusercontent.com/8490199/75513431-9cb06980-59d3-11ea-97e3-5dbf11b8a304.png">


#### Does it require any deployments updates ?
